generate-types.js

```diff
@@ -0,0 +1,60 @@
+const TypeWriter = require('@gimenete/type-writer')
+const webhooks = require('@octokit/webhooks-definitions/index.json')
+const pascalCase = require('pascal-case')
+const prettier = require('prettier')
+const fs = require('fs')
+
+const types = []
+const signatures = []
+for (const webhook of webhooks) {
+  const { name, actions, examples } = webhook
+  if (!examples) continue
+  const tw = new TypeWriter()
+  const typeName = `WebhookPayload${pascalCase(name)}`
+  tw.add(examples, { rootTypeName: typeName })
+  const events = [
+    `'${name}'`,
+    ...actions.map(action => `'${name}.${action}'`)
+  ].join(' | ')
+  types.push(`type ${typeName} = ${tw.generate('typescript', { inlined: true })}`)
+  signatures.push(`
+    public on (event: ${events}, callback: (event: Webhooks.WebhookEvent<${typeName}>) => void)
+  `)
+}
+
+const definition = `
+// import { Application } from 'express'
+
+interface Options {
+  path: string,
+  secret: string
+}
+
+${types.join('\n\n')}
+
+declare class Webhooks {
+  // public middleware: Application
+
+  constructor (Options)
```

Comment:
Ref: probot/probot#798
```suggestion
  constructor (options: Options)
```

Reply:
@gimenete You seem to have removed this change in one of your commits